Sources/Drill/Invocation.swift

```diff
@@ -67,12 +67,11 @@ public struct Invocation {
           SyntaxDumper(stream: &stderrStream).dump(module)
         }
       case .dump(.scopes):
-//        let layoutTokens = layout(tokens)
-//        let parser = Parser(diagnosticEngine: engine, tokens: layoutTokens)
-//        let module = parser.parseTopLevelModule()!
-//        let binder = NameBinding(topLevel: module, engine: engine)
-//        print(binder.performScopeCheck(topLevel: module))
-        break
+        let layoutTokens = layout(tokens)
```

Comment:
I feel like we need to encapsulate the set of passes here because we keep copying `lex->layout->parse` everywhere...